src/snowflake/snowpark/modin/plugin/compiler/snowflake_query_compiler.py

```diff
@@ -17507,3 +17507,118 @@ def tz_convert(self, *args: Any, **kwargs: Any) -> None:
 
     def tz_localize(self, *args: Any, **kwargs: Any) -> None:
         ErrorMessage.method_not_implemented_error("tz_convert", "BasePandasDataset")
+
+    def create_qc_with_index_data_and_qc_index(
```

Comment:
Is the meaning of the function name that the `data` passed to a frame/series constructor is a lazy index, and `index` is a query compiler?

Reply:
yes!